libminifi/src/utils/BackTrace.cpp

```diff
@@ -117,6 +117,8 @@ void pull_trace(uint8_t frames_to_skip /* = 1 */) {
 
     TraceResolver::getResolver().addTraceLine(file_name, symbol_name.c_str(), symbol_offset);
   }
+#else
+  (void)(frames_to_skip);
```

Comment:
It is hard to see what condition this `#else` refers to.  I think
```c++
#ifndef HAS_EXECINFO
void pull_trace(uint8_t) {}
#else
void pull_trace(uint8_t frames_to_skip /* = 1 */) {
...
}
#endif
```
would be more readable.

Reply:
I agree. Because indentation cannot be used to mark nested `#ifdef`s I added some more comments to help a bit.